interfaces/builtin/broadcom_asic_control.go

```diff
@@ -38,12 +38,20 @@ const broadcomAsicControlConnectedPlugAppArmor = `
 /dev/linux-user-bde rw,
 /dev/linux-kernel-bde rw,
 /dev/linux-bcm-knet rw,
+
+# These are broader than they needs to be, but until we query udev
+# for specific devices, use a broader glob
```

Comment:
Indeed, that looks way too broad. In fact, a bit too broad to be right. Not sure it makes sense to turn the "broadcoam-asic" interface in a sort of "read about every single PCI device in the system".

@jdstrand Note that we won't be able to change that in the future without the potential of breaking clients unknowingly.

What's the plan here?